ugvc/pipelines/mrd/srsnv_training.py

```diff
@@ -83,6 +83,12 @@ def parse_args(argv: list[str]) -> argparse.Namespace:
         type=str,
         help="""comma separated list of categorical features for ML classifier """,
     )
+    parser.add_argument(
+        "--balanced_sampling_info_fields",
+        type=str,
+        help="comma separated list of categorical features to be used for balanced sampling of the TP training set"
+        " to eliminate prior distribution bias (e.g. 'trinuc_context_with_alt,is_forward')",
```

Comment:
remind me, the context of forward and reverse reads on the same snv is the same or revcom?

Reply:
The same, that's why I added is_forward

Reply:
So your intent is that in the end we will the same number of +/- reads per motif?
I'm not sure why wouldn't it occur naturally in germline variants.